django_cron/cron.py

```diff
@@ -7,48 +7,94 @@
 
 class FailedRunsNotificationCronJob(CronJobBase):
     """
-        Send email if cron failed to run X times in a row
+    A regular job to send email reports for failed Cron jobs.
+
+    The job log is used to check for all unreported failures for each job
+    classes specified within the CRON_CLASSES dictionary. When the number of
+    failures for each job type exceeds the limit (which can be specified
+    either per-job or project wide) an email is sent to all relevant parties
+    detailing the error.
     """
-    RUN_EVERY_MINS = 30
+    RUN_EVERY_MINS = 0
 
     schedule = Schedule(run_every_mins=RUN_EVERY_MINS)
     code = 'django_cron.FailedRunsNotificationCronJob'
 
     def do(self):
+        self.config = self.get_config()
+        cron_classes = [get_class(c_name) for c_name in settings.CRON_CLASSES]
```

Comment:
Be as explicit as possible throughout. `c_name` to `class_name` , `cron_cls` to `cron_class` (this is referenced a lot), and avoid single letter variables.  While it isn't difficult to follow it would be nice.